networking/p2p/src/test/java/tech/pegasys/artemis/networking/p2p/jvmlibp2p/gossip/BlocksTopicHandlerTest.java

```diff
@@ -52,6 +54,7 @@
 
   @BeforeEach
   public void setup() {
+    doReturn(CompletableFuture.completedFuture(null)).when(publisher).publish(any(), any());
```

Comment:
What does this line do here?

Reply:
(not criticizing, I just want to understand what it does)

Reply:
I added logging based on the returned `CompletableFuture` from `publisher.publish()`: https://github.com/PegaSysEng/artemis/pull/986/files#diff-c9c5dcd30aee323633d9772415dc79baR94

So, the publisher mock needs to return something, otherwise there's a null pointer exception.  This line tells it return a successful future when invoked with any arguments.

Reply:
thank you for the explanation.